uaclient/security_status.py

```diff
@@ -156,6 +160,10 @@ def security_status(cfg: UAConfig) -> Dict[str, Any]:
             }
         )
 
+    summary["num_installed_packages_main"] = package_count["main"]
+    summary["num_installed_packages_restricted"] = package_count["restricted"]
+    summary["num_installed_packages_universe"] = package_count["universe"]
+    summary["num_installed_packages_multiverse"] = package_count["multiverse"]
```

Comment:
I think we should be safe here and just change to a `get` call instead, just to guarantee that even on odd scenarios, we will  not error out because of a missing key

Reply:
It is a `defaultdict(int)`, so it will return `0` for any missing key. This is covered in the tests, given there are no packages mocked there to be from `universe` or `multiverse` for instance, and those keys are present in the output